### DISCUSSION TITLE
Uzun dizinin terimleri sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#36] [Soru Cevap] [answered]
Şöyle bir sayı dizisi düşünelim. İlk terim 20 dir, bundan sonraki her terim kendinden önceki terimin karesinin rakamları toplamına 1 eklenerek bulunur.
(2. terim 5 tir çünkü (20)^2 = 400 ve 4+0+0= 5 tir)
Bu dizinin 100. terimi nedir.

**Comment by OmerAkbayrak** [comment, marked as answer]
**Bu sayı dizisinin son terimi 11'dir. Çünkü sayı dizisinin ilerleyişi : 20, 5, 8, 11, 5, 8, 11  şeklinde, ve bu ilerleyişte 5, 8 ve 11 sayıları tekrar ediyor bu yüzden 100. terimi bulmak için 100'den tekrar etmeyen terim sayısını çıkarırsak  99 kalır ve 99'u 3'e böldüğümüzde kalan 0 olur bu yüzden         döngüdeki son eleman olan 11, 100. eleman olacaktır.**
